includes/User.php

```diff
@@ -671,7 +671,7 @@ public static function isCreatableName( $name ) {
 
 		// Preg yells if you try to give it an empty string
 		if( $wgInvalidUsernameCharacters !== '' ) {
-			if( preg_match( '/[' . preg_quote( $wgInvalidUsernameCharacters, '/' ) . ']/', $name ) ) {
+			if( preg_match( $wgInvalidUsernameCharacters, $name ) ) {
 				wfDebugLog( 'username', __METHOD__ .
```

Comment:
You might want to consider logging it using WikiaLogger in order to analyze number of occurences